docs/.vale/thothtech/InclusionGender.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Suggestion: thothtech.InclusionGender
+#
+# Suggests alternatives for words that are gender-specific.
+#
+# For a list of all options, see https://errata-ai.gitbook.io/vale/getting-started/styles
+extends: substitution
+message: 'Use inclusive language. Consider "%s" instead of "%s".'
+link: TBA
+level: suggestion
+ignorecase: true
+swap:
+  mankind: humanity, people
```

Comment:
**Suggestion:** Potential to add manpower into the suggestion list as workforce or something of the sort

Reply:
Great idea. Incorporated.